pom.xml

```diff
@@ -35,7 +35,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <bullet.core.version>0.2.5</bullet.core.version>
+        <bullet.core.version>0.3.0</bullet.core.version>
```

Comment:
Can we also bump this project from 0.2.1-SNAPSHOT to 0.3.0-SNAPSHOT since this breaks interface

Reply:
Done.